src/components/datalist-item/__tests__/DatalistItem.test.tsx

```diff
@@ -1,66 +1,44 @@
 import React from 'react';
 import sinon from 'sinon';
-
+import { shallow } from 'enzyme';
```

Comment:
This is weird, did the tests even work before when this wasn't imported?

Reply:
No idea, my IDE(intellij) flagged it as missing, I dont know how this was working before.